code/game/objects/items/weapons/explosives.dm

```diff
@@ -35,6 +35,8 @@
 				message_say = "FOR NAR-SIE!"
 			else if(role == "revolutionary" || role == "head revolutionary")
 				message_say = "VIVA LA REVOLUTION!"
+			else if(user.mind.gang_datum)
+				message_say = "[uppertext(user.mind.gang_datum.name)] GANG RULES!"

```

Comment:
wouldn't this cause things like "tunnel gang gang rules!"?
why not just make it say "X RULES!" instead of "X GANG RULES!"